src/helpers/dateGenerator.js

```diff
@@ -0,0 +1,3 @@
+export function dateGenerator(date) {
+	return date.replaceAll('/', '.');
```

Comment:
'10/11/2020' - this is MM/DD/YYYY format, could you modify dateGenerator function so that date '8/3/2021' would look like 03.08.2021 and not 8.3.2021

Reply:
But isn't 8/3/2021 already in dd/mm/yyyy format?
![image](https://user-images.githubusercontent.com/30635536/152643127-9033f0d7-d479-4cef-b558-26a4048c0a72.png)
I suppose I only need to change date format when created in CreateCourse.